python/tk_multi_workfiles/actions/context_change_action.py

```diff
@@ -0,0 +1,67 @@
+# Copyright (c) 2015 Shotgun Software Inc.
+#
+# CONFIDENTIAL AND PROPRIETARY
+#
+# This work is provided "AS IS" and subject to the Shotgun Pipeline Toolkit
+# Source Code License included in this distribution package. See LICENSE.
+# By accessing, using, copying or modifying this work you indicate your
+# agreement to the Shotgun Pipeline Toolkit Source Code License. All rights
+# not expressly granted therein are reserved by Shotgun Software Inc.
+
+"""
+Action to create a new file.
+"""
+
+from sgtk import TankError
+from sgtk.platform.qt import QtGui
+
+from .file_action import FileAction
+from .action import Action
+
+
+class ContextChangeAction(Action):
+    """
+    This action changes the current engine context.
+    """
+
+    def __init__(self, environment):
+        """
+        Constructor.
+
+        :param environment: A WorkArea instance containing the context we will switch to.
+        """
+        Action.__init__(self, "Change Context")
+        self._environment = environment
+
+    def execute(self, parent_ui):
+        """
+        Perform a new-scene operation initialized with the current context.
```

Comment:
```suggestion
        Perform a context change operation.
```

Reply:
thats annoying I did a pass on this file, to make sure I'd renamed it. It was obviously a bad pass :(